build.gradle

```diff
@@ -2,47 +2,39 @@ buildscript {
     repositories {
         jcenter()
     }
-
     dependencies {
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
-        classpath "io.spring.gradle:dependency-management-plugin:0.5.5.RELEASE"
-        classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:3.0.3'
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.5.RELEASE")
+        classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:2.2.+'
     }
 }
 
 
 allprojects {
     group = 'info.developerblog.spring.thrift'
-    version = '0.10.0'
+    version = '1.0.0.RC1'
 
     repositories {
         jcenter()
     }
-    
+
     apply plugin: 'idea'
-    apply plugin: "io.spring.dependency-management"
-    apply plugin: 'nebula.provided-base'
+    apply plugin: 'io.spring.dependency-management'
+    apply plugin: 'provided-base'
 
     dependencyManagement {
-        dependencies {
-            dependencySet(group:'org.springframework.boot', version: '1.3.2.RELEASE') {
-                entry "spring-boot-starter-web"
-                entry "spring-boot-starter-test"
-                entry "spring-boot-starter-actuator"
-            }
-
-            dependencySet(group:'org.springframework.cloud', version: '1.0.6.RELEASE') {
-                entry "spring-cloud-starter-ribbon"
-            }
-
-            dependency "org.apache.thrift:libthrift:0.9.2"
-
-            dependency "org.codehaus.groovy:groovy-all:2.4.4"
+        imports {
+            mavenBom "org.springframework.cloud:spring-cloud-starter-parent:Brixton.RELEASE"
+        }
 
-            dependency "org.projectlombok:lombok:1.16.2"
+        dependencies {
+            dependency 'org.projectlombok:lombok:1.16.8'
```

Comment:
We've been removing lombok in some of our libraries cause it could make it difficult for the users to contribute. Just saying...